core/examples/luigi-core-material-ui/src/components/Header.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```

Comment:
```suggestion
import React, { Fragment } from 'react';
```
This change is up to you.

Reply:
As a newbie to react, I have the feeling it is simpler to see what is coming from React and what is internal if it is prefixed.